custom/icds_reports/utils/data_pull.sql

```diff
@@ -28,6 +28,31 @@ CREATE TABLE "temp_visit_table" AS (
 --                                  Filter: ((location_entered IS NOT NULL) AND (submitted_on >= '2020-02-01'::date) AND (submitted_on < '2020-04-01'::date) AND (location_entered <> ''::text))
 -- (13 rows)
 
+CREATE TABLE "temp_visit_table_april" AS (
+    SELECT
+        location_id as supervisor_id,
+        COUNT (DISTINCT awc_id) as unique_visits_april_month
+    FROM "ucr_icds-cas_static-awc_mgt_forms_ad1b11f0"
+    WHERE
+        submitted_on>= '2020-03-01'
```

Comment:
this will let march forms be considered not April. SImilarly, submitted_on filter needs to changed in next query.

Reply:
updated, not planning to move the query to management command (as for this data pull) as the management command is working fine, will add if we add to out codebase

Reply:
do you know is this requirement is frozen or expected to change?